ports/unix/README.md

```diff
@@ -1,14 +1,49 @@
-The Unix version
-----------------
+# The Unix port
 
-The "unix" port requires a standard Unix-like environment with gcc and GNU
-make. This includes Linux, BSD, macOS, and Windows Subsystem for Linux. The
+The "unix" port requires a standard Unix-like environment with gcc, GNU
+make, and Python 3.x. This includes Linux, BSD, macOS, and Windows Subsystem for Linux. The
 x86 and x64 architectures are supported (i.e. x86 32- and 64-bit), as well as
 ARM and MIPS. Making a full-featured port to another architecture requires
 writing some assembly code for the exception handling and garbage collection.
 Alternatively, a fallback implementation based on setjmp/longjmp can be used.
 
-To build (see section below for required dependencies):
+# Building
+
+## Dependencies
+
+To build the "minimal" variant you will need:
+
+* A git client, and a local git clone of the MicroPython source
+* gcc (or clang for macOS) toolchain
+* Python 3.x
+
+To build the default variant and configuration, then you will also need:
+
+* `pkg-config` tool
+* `libffi` library and headers
+
+On Debian/Ubuntu/Mint and related distros, you can install all these
+dependencies with a command like:
+
+```
+# apt install build-essential git python3 pkg-config libbfi-dev
+```
+
+(To build either standalone or minimal MicroPython executables that don't
+require system `libffi` or `pkg-config`, see below.)
+
+## Building mpy-cross
+
+Build the [MicroPython
+cross-compiler](https://github.com/micropython/micropython/?tab=readme-ov-file#the-micropython-cross-compiler-mpy-cross):
+
+    $ cd mpy-cross
```

Comment:
Maybe mention this needs to be done from the top level of this repo?

Reply:
I liked the way you put it in the qemu PR, so have updated along those lines. PTAL.

Reply:
Great!